samples/nvidia-resnet/troubleshooting.md

```diff
@@ -0,0 +1,62 @@
+# Common issues and Troubleshooting
+
+Kubernetes provides several useful tools for debugging pipelines. To continiously monitor the status of all Kubeflow pods and services, one can do:
+
+`watch kubectl -n kubeflow get all `
+
+```
+pod/profiles-6b5f999584-pjj7j                                   1/1     Running            1          86m
+pod/pytorch-operator-577469ccd8-p8r4z                           1/1     Running            1          86m
+pod/resnet-cifar10-pipeline-2f6v8-4205778421                    0/2     Error              0          40m
+pod/resnet-cifar10-pipeline-5pz5w-2553990163                    0/2     Completed          0          4m
+pod/resnet-cifar10-pipeline-5pz5w-2832477042                    0/2     Completed          0          29m
+pod/resnet-cifar10-pipeline-5pz5w-349525310                     0/2     Completed          0          25m
+pod/resnet-cifar10-pipeline-5pz5w-4048043379                    0/2     Completed          0          3m52s
+pod/resnet-cifar10-pipeline-t2kv2-4047898736                    0/2     Error              0          82m
+```
+
+Pipelines with error status can be inspected with:
+
+`kubectl logs   resnet-cifar10-pipeline-t2kv2-4047898736 --namespace kubeflow  --all-containers`
+
+To continuously monitor a pipeline:
+
+`kubectl logs -f   resnet-cifar10-pipeline-t2kv2-4047898736 --namespace kubeflow  --all-containers`
+
+## 1. Kubeflow ambassador stuck in a crash loop.
```

Comment:
After this line, it's for kubeflow troubleshooting. Before this line, they can be KFP tsg.